### PR TITLE
[Inflector][String] wrong plural form of words ending by "pectus"

### LINKED ISSUE #41124: [String] wrong plural form of "conspectus"
**Symfony version(s) affected**: 5.2.6

**Description**
Pluralize method of EnglishInflector said plural of conspectus is conspecti.
As you can see the correct plural form is conspectuses:
https://en.wiktionary.org/wiki/conspectus

**How to reproduce**
```
<?php

// composer require symfony/string

require 'vendor/autoload.php';

use Symfony\Component\String\Inflector\EnglishInflector;

$inf = new EnglishInflector();

var_dump($inf->pluralize('conspectus')[0] === 'conspectuses'); // false
var_dump(count($inf->pluralize('conspectus'))); // 1
```

### PR DESCRIPTION
**Pull request by makraz**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #41124
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Component/Inflector/Inflector.php [hunk 1]
```diff
@@ -45,6 +45,9 @@ final class Inflector
         // geese (goose)
         ['esee', 4, false, true, 'oose'],
 
+        // conspectuses (conspectus), prospectuses (prospectus)
+        ['sesutcep', 8, true, true, 'pectus'],
```
Comment (Nyholm) [inline src/Symfony/Component/Inflector/Inflector.php]: Could you put this in alphabetical order? Ie below `seivom`?

File: src/Symfony/Component/Inflector/Inflector.php [hunk 2]
```diff
@@ -269,6 +272,9 @@ final class Inflector
         // circuses (circus)
         ['suc', 3, true, true, 'cuses'],
 
+        // conspectuses (conspectus), prospectuses (prospectus)
+        ['sutcep', 6, true, true, 'pectuses'],
```
Comment (Nyholm) [inline src/Symfony/Component/Inflector/Inflector.php:276]: if this line is removed, will the tests still pass?

**Review by Nyholm — COMMENTED** [review]
Thank you. 

I just have two small questions

File: src/Symfony/Component/Inflector/Inflector.php (hunk 2 above)
Comment (makraz) [reply src/Symfony/Component/Inflector/Inflector.php:276]: no, the tests failed
image

**Comment by nicolas-grekas** [comment]
Thank you @makraz.

**Pull request merged**